integration_tests/testsuite/test_monitoring.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+
+import json
+import logging
+import requests
+import time
+
+import test_util
+
+from google.cloud import monitoring as gcloud_monitoring
+
+
+def _test_monitoring(base_url):
```

Comment:
these are integration tests though, which in some regards are way out of the scopes of python's unittest framework (for example, tests aren't parameterizable by design, which I need). does it still make sense to try and build these tests on top of it? maybe something like Pytest is more appropriate?